src/gui/wizard/owncloudwizard.cpp

```diff
@@ -245,4 +246,22 @@ AbstractCredentials *OwncloudWizard::getCredentials() const
     return 0;
 }
 
+bool OwncloudWizard::askExperimentalPlaceholderFeature()
+{
+    QMessageBox msgBox(
+        QMessageBox::Warning,
+        tr("Enable experimental feature?"),
+        tr("When the \"synchronize placeholders\" mode is enabled no files will be downloaded initially. "
+           "Instead, a tiny \".owncloud\" file will be created for each file on the server. "
+           "The contents can be downloaded by running these files or by using their context menu."
+           "\n\n"
+           "This is a new, experimental mode. If you decide to use it, please report any "
+           "issues that come up."));
+    QPushButton *yesBtn = msgBox.addButton(tr("Enable experimental mode"), QMessageBox::DestructiveRole);
```

Comment:
Interrestingly, pressing "Enter" still enable the box.  However, pressing Esc has no effect.

I would not think that the experimental mode is destructive, as one can still uncheck the box.
I would say it is more intuitive if "esc" is the same as "stay save" and "Enter"  the same as enable